lgsm/functions/core_functions.sh

```diff
@@ -121,12 +121,15 @@ fn_fetch_function
 }
 
 command_postdetails.sh(){
-functionfile="${FUNCNAME}"
-tempffname=$functionfile
-fn_fetch_function
-functionfile="command_details.sh"
-fn_fetch_function
-functionfile=$tempffname
+    functionfile="${FUNCNAME}"
+    tempffname=$functionfile
+    # First, grab the command_postdetails.sh file
+    fn_fetch_function
+    # But then next, command_details.sh needs to also be pulled
+    # because command_postdetails.sh sources its functions -CedarLUG
```

Comment:
It is - tempffname stores command_postdetails.sh.

The gist of the code here is that we need command_details.sh to be automatically pulled as a dependency of command_postdetails.sh.  So the first fn_fetch_function pulls the command_postdetails.sh as expected (line 126), but then I need to also pull command_details.sh (because it's sourced by command_postdetails.sh) on line 128.  

Before returning, functionfile is reset back to command_postdetails.sh.

The usage case here is if the user has nothing in lgsm/functions and issues `./scriptname pd`,
it will pull command_postdetails.sh correctly, but die because command_details.sh wasn't present.

So lines 125-128 are making sure that command_details.sh is present as a dependency whenever command_postdetails.sh is invoked.